README.md

```diff
@@ -277,6 +277,147 @@ MotionAnimator is a purely Core Animation-based animator. If you are looking for
 - [POP](https://github.com/facebook/pop)
 - [CADisplayLink](https://developer.apple.com/documentation/quartzcore/cadisplaylink)
 
+# Core Animation: a deep dive
+
+> Recommended reading:
+>
+> - [Building Animation Driven Interfaces](http://asciiwwdc.com/2010/sessions/123)
+> - [Core Animation in Practice, Part 1](http://asciiwwdc.com/2010/sessions/424)
+> - [Core Animation in Practice, Part 2](http://asciiwwdc.com/2010/sessions/425)
+> - [Building Interruptible and Responsive Interactions](http://asciiwwdc.com/2014/sessions/236)
+> - [Advanced Graphics and Animations for iOS Apps](http://asciiwwdc.com/2014/sessions/419)
+> - [Advances in UIKit Animations and Transitions](http://asciiwwdc.com/2016/sessions/216)
+> - [Animating Layer Content](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/CreatingBasicAnimations/CreatingBasicAnimations.html)
+> - [Advanced Animation Tricks](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/AdvancedAnimationTricks/AdvancedAnimationTricks.html)
+> - [Additive animations: animateWithDuration in iOS 8](http://iosoteric.com/additive-animations-animatewithduration-in-ios-8/)
+
+There are two primary ways to animate with Core Animation on iOS:
+
+1. **implicitly**, with the UIView `animateWithDuration:` APIs, or by setting properties on standalone CALayer instances (those that are **not** backing a UIView), and
```

Comment:
Is this not still wrong?

This example implicitly animates a property.
```swift
let view = UIView()
window.addSubview(view)
let layer = CALayer()
view.layer.addSublayer(layer)

// Pump the run loop once.
RunLoop.main.run(mode: .defaultRunLoopMode, before: .distantFuture)

layer.opacity = 0.5
```

Reply:
Oh I guess it isn't backing the view. Ignore me.

Reply:
To make this comment thread useful does the following implicitly animate?

```swift
let view = UIView()
window.addSubview(view)

// Pump the run loop once.
RunLoop.main.run(mode: .defaultRunLoopMode, before: .distantFuture)

view.layer.opacity = 0.5
```

Reply:
It would not.